### PR TITLE
Add model identifier.

### PR DESCRIPTION
**Pull request by regadas**

### CODE REVIEW

**Comment by regadas** [comment]
@ravwojdyla @andrewsmartin PTAL

File: zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowGraphModel.java [hunk 1]
```diff
@@ -45,21 +45,41 @@
 
   private static final Logger logger = LoggerFactory.getLogger(TensorFlowGraphModel.class);
 
+  static final Model.Id ID = Id.create("tensorflow-graph");
```
Comment (andrewsmartin) [inline zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowGraphModel.java]: Should we rename this `DEFAULT_ID`?

File: zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowModel.java [hunk 2]
```diff
@@ -38,6 +38,8 @@
 @AutoValue
 public abstract class TensorFlowModel implements Model<SavedModelBundle> {
 
+  static final Model.Id ID = Id.create("tensorflow");
```
Comment (andrewsmartin) [inline zoltar-tensorflow/src/main/java/com/spotify/zoltar/tf/TensorFlowModel.java]: Same as above, `DEFAULT_ID`?

File: zoltar-xgboost/src/main/java/com/spotify/zoltar/xgboost/XGBoostModel.java [hunk 3]
```diff
@@ -40,17 +40,27 @@
 @SuppressWarnings("checkstyle:AbbreviationAsWordInName")
 @AutoValue
 public abstract class XGBoostModel implements Model<Booster> {
+  static final Model.Id ID = Id.create("xgboost");
```
Comment (andrewsmartin) [inline zoltar-xgboost/src/main/java/com/spotify/zoltar/xgboost/XGBoostModel.java]: `DEFAULT_ID`?

**Pull request merged**